repo/packages/H/hdfs/0/config.json

```diff
@@ -8,7 +8,7 @@
         "type": "string"
       },
       "default": [
-        "https://downloads.mesosphere.io/hdfs/dcos/0.1.0/hdfs-mesos-0.1.0.tgz"
+        "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/dcos-ea/0.1.0/hdfs-mesos-0.1.0.tgz"
```

Comment:
Did the software change, or is it just configuration?  If there were source code modifications we should rev the version instead (doesn't look like a snapshot, so assuming the tag is immutable).